.travis.yml

```diff
@@ -1,9 +1,9 @@
 language: ruby
 
 rvm:
-  - 2.2
   - 2.3
-  - 2.4.1
+  - 2.4
```

Comment:
I don't think we need to test all supported versions of ruby.  I think that is only necessary if we have ruby version specific code.  I think we only need to test the latest ruby version and the minimum supported ruby version.

Reply:
2.4 testing is now removed